ecosystem/README.md

```diff
@@ -10,6 +10,7 @@
   and it requires approval by 2 SDF members of the SEP Team.
 
 ## SEP Status Terms
+* **Deprecated** - A SEP that was previously on an active track but has been deprecated and is no longer suggested for use. There may be legacy usage of a deprecated SEP.
```

Comment:
I think we should move this into the `Additional Statuses` section. The statuses in the `SEP Status Terms` section are for proposals that are accepted or on track for being accepted. The statuses in the `Additional Statuses` appear to be for proposals that are no longer in use, which a deprecated proposal is more related.

Reply:
I agree: this should move to `Additional Statuses`.  The current `Status Terms` list is in order of evolution from Draft through Final.